css/amp.css

```diff
@@ -410,3 +410,7 @@ amp-live-list > [update] {
 amp-experiment, amp-share-tracking {
   display: none;
 }
+
+amp-fresh {
+  visibility: hidden;
```

Comment:
Please add a comment here. E.g. "amp-fresh starts as invisible by default and is shown once the content is updated. Can be customized by the page author."

And please make sure to start a thread with me and Eric to confirm if this is what we want as a default behavior.